src/App.js

```diff
@@ -0,0 +1,85 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { FiTrash, FiPlus } from "react-icons/fi";
+
+function App() {
+
+  const [todos, setTodos] = useState(JSON.parse(localStorage.getItem("todos")) || []);
+  const [newTask, setNewTask] = useState("");
+
+  function addTodos(task) {
+    const todo = { id: nanoid(), task, completed: false };
+    setTodos((prevTodos) => {
+      const newTodo = [todo, ...prevTodos];
+      localStorage.setItem("todos", JSON.stringify(newTodo));
+      return newTodo;
+    });
+  }
+
+  function complete(todoId) {
+    const updatedTask = todos.map((x) =>
+      x.id === todoId ? { ...x, completed: !x.completed } : x
+    );
+    localStorage.setItem("todos", JSON.stringify(updatedTask));
+    setTodos(updatedTask);
+  }
+
+  function deleteTodos(todoId) {
+    const deleteTask = todos.filter((x) => x.id !== todoId);
+    localStorage.setItem("todos", JSON.stringify(deleteTask));
+    setTodos(deleteTask);
+  }
+
+  return (
+    <div>
+      <div className="flex flex-col justify-center items-center h-screen bg-pink-300 overflow-y-scroll">
+        <h1 className="text-green-900 opacity-30 text-9xl content-center">
+          todos
+        </h1>
+        <div className="flex items-center border border-gray-200 rounded-lg h-12 w-1/3 mt-10 bg-slate-100 text-black">
+          <input
+            onChange={(e) => setNewTask(e.target.value)}
+            type="search"
+            className="w-full px-5 bg-transparent outline-none"
+            placeholder="Add todo..."
+          />
+          <button
+            onClick={() => addTodos(newTask)}
+            className="text-white rounded-r-lg w-12 flex justify-center items-center bg-green-500 h-full border border-green-500 hover:bg-green-500 focus:ring-4 focus:outline-none focus:ring-green-500 dark:bg-green-500 dark:hover:bg-green-500 dark:focus:ring-green-500"
+          >
+            <FiPlus />
+          </button>
+        </div>
+        {todos.map((_todo, index) => {
+          return (
+            <div key={index} className="flex justify-center items-center border-b-2 border-green-300 mt-10 pb-2">
```

Comment:
Using index as the key is not the best practice.
```suggestion
            <div key={_todo.id} className="flex justify-center items-center border-b-2 border-green-300 mt-10 pb-2">
```